### PR TITLE
[master] Add python 3.13 requirements files

### PR DESCRIPTION
**Pull request by dwoz**
### What issues does this PR fix or reference?
Fixes

### Previous Behavior
Remove this section if not relevant

### New Behavior
Remove this section if not relevant

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html) for best practices, including the
[PR Guidelines](https://docs.saltproject.io/en/master/topics/development/pull_requests.html).

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Review by dmurphy18 — DISMISSED** [review]
Wondering about some of the dependencies, for example: cryptography does state support for Py 3.13 yet but presume it works.
Can we drop Py 3.8 given it is EOL as of beginning of this month, and will be really dead by the time this hits the streets.
Also do we still need to keep freebsd around, given we are no longer supporting it, or is it good to at least have it there for Krill etc., make his life easier.

**Comment by dwoz** [comment]
> Wondering about some of the dependencies, for example: cryptography does state support for Py 3.13 yet but presume it works. Can we drop Py 3.8 given it is EOL as of beginning of this month, and will be really dead by the time this hits the streets. Also do we still need to keep freebsd around, given we are no longer supporting it, or is it good to at least have it there for Krill etc., make his life easier.

I actually think we should drop support for anything less than 3.10 in 3008.

**Comment by dmurphy18** [comment]
@dwoz I'd second that, dropping support for everything less than Python 3.10

File: requirements/base.txt [hunk 1]
```diff
@@ -38,7 +40,7 @@ gitpython>=3.1.37; sys_platform == 'win32'
 lxml>=4.6.3; sys_platform == 'win32'
 pymssql>=2.2.1; sys_platform == 'win32'
 pymysql>=1.0.2; sys_platform == 'win32'
-pythonnet>=3.0.1; sys_platform == 'win32'
+pythonnet>=3.0.0; sys_platform == 'win32'
```
Comment (twangboy) [inline requirements/base.txt:43]: It's interesting why this works:
https://pypi.org/project/pythonnet/#history

**Review by twangboy — DISMISSED** [review]

**Review by dmurphy18 — DISMISSED** [review]
A lot of the requirements will change once we get the deprecated files out there, for example: pyvmomi from common.in, but guess got to live with it for now.
Get rid of Py 3.8 and Py 3.9 in a different PR, I suppose

File: requirements/base.txt (hunk 1 above)
Comment (dwoz) [reply requirements/base.txt:43]: Yeah, we shouldn't proceed like this. I'll add a constraint for 3.13.

**Pull request merged**